packages/web-app-admin-settings/tests/unit/views/Users.spec.ts

```diff
@@ -72,37 +139,107 @@ describe('Users view', () => {
   describe('method "editUser"', () => {
     it('should emit event on success', async () => {
       const editUser = {
+        appRoleAssignments: [
+          {
+            appRoleId: '2',
+            resourceId: 'some-graph-app-id',
+            principalId: '1'
+          }
+        ],
+        displayName: 'administrator',
         id: '1',
-        displayName: 'jan',
-        role: { id: '1', displayName: 'admin' },
-        drive: { id: '1', quota: { total: 10000 } },
-        passwordProfile: { password: 'newpassword' }
+        mail: 'administrator@example.org',
+        drive: {
+          id: '1',
+          name: 'admin',
+          quota: {
+            total: 1000000000
+          }
+        },
+        passwordProfile: {
+          password: 'administrator'
+        }
       }
 
-      const graph = defaultGraphMock()
-      graph.users.editUser.mockImplementation(() =>
+      const graph = getDefaultGraphMock()
+      graph.users.editUser.mockImplementation(() => mockAxiosResolve())
+      graph.users.createUserAppRoleAssignment.mockImplementation(() => mockAxiosResolve())
+      graph.drives.updateDrive.mockImplementation(() =>
         mockAxiosResolve({
-          accountUuid: '1',
           id: '1',
-          roleId: '1'
+          name: 'admin',
+          quota: { remaining: 1000000000, state: 'normal', total: 1000000000, used: 0 }
         })
       )
-      graph.drives.updateDrive.mockImplementation(() => mockAxiosResolve({ name: 'any' }))
-      const { wrapper } = getMountedWrapper({ graph })
+      graph.users.getUser.mockImplementation(() =>
+        mockAxiosResolve({
+          appRoleAssignments: [
+            {
+              appRoleId: '2',
+              id: '1',
+              principalId: '1',
+              principalType: 'User',
+              resourceDisplayName: 'ownCloud Infinite Scale',
+              resourceId: 'some-graph-app-id'
+            }
+          ],
+          displayName: 'administrator',
+          drive: {
+            id: '1',
+            name: 'admin',
+            quota: { remaining: 1000000000, state: 'normal', total: 1000000000, used: 0 }
+          },
+          id: '1',
+          mail: 'administrator@example.org',
+          memberOf: [],
+          onPremisesSamAccountName: 'admin',
+          surname: 'Admin'
+        })
+      )
+
+      const { wrapper } = getMountedWrapper({
+        graph,
+        data: {
+          selectedUsers: [
+            {
+              ...getDefaultUser(),
+              quota: {
+                remaining: 1000000000,
+                state: 'normal',
+                total: 1000000000,
+                used: 0
+              }
+            }
+          ]
```

Comment:
Setting the `selectedUsers` via `data` won't work anymore, because the property is not included in `data` anymore (in `Users.vue`). It's now declared via `const selectedUsers = ref([])`.

You need to either set it the "correct way" by interacting with the UI or the "dirty way" via `wrapper.vm.selectedUsers = []`, followed by a `await wrapper.vm.$nextTick()`.

... yeah I know, that's super annoying :(

You can remove `data` here and also in `getMountedWrapper` then.